runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogData.java

```diff
@@ -32,14 +31,20 @@
     @Getter
     byte[] data;
 
-    private ByteBuf serializedCache = null;
+    private SerializedCache serializedCache = null;
```

Comment:
I think `serializedCache` should be Optional<SerializedCache> it will make code more secure and less error-prone